code/lab/dialogs/lab_ui_helpers.cpp

```diff
@@ -3,6 +3,22 @@
 #include "cfile/cfile.h"
 #include "lab/labv2_internal.h"
 
+SCP_map<int, SCP_string> get_docking_point_map(int model_index)
+{
+	SCP_map<int, SCP_string> result;
+
+	polymodel* pm = model_get(model_index);
+	if (pm == nullptr || pm->n_docks <= 0)
+		return result;
+
+	for (int i = 0; i < pm->n_docks; ++i) {
+		const char* name = pm->docking_bays[i].name;
+		result[i] = (name && *name) ? SCP_string(name) : SCP_string("<unnamed>");
```

Comment:
Any value in making an index to be added to `<unnamed>` in case of a long list of unnamed docking bays?

Reply:
Won't hurt!